core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -48,6 +48,26 @@
    Map<Address, Response> invokeRemotely(Map<Address, ReplicableCommand> rpcs, RpcOptions options);
 
    /**
+    * Asynchronously sends the {@link ReplicableCommand} to the destination using the specified {@link DeliverOrder}.
+    *
+    * @param destination  the destination's {@link Address}.
+    * @param command      the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    */
+   void sendTo(Address destination, ReplicableCommand command, DeliverOrder deliverOrder);
+
+   /**
+    * Asynchronously sends the {@link ReplicableCommand} to the set of destination using the specified {@link
+    * DeliverOrder}.
+    *
+    * @param command      the {@link ReplicableCommand} to send.
+    * @param deliverOrder the {@link DeliverOrder} to use.
+    * @param destinations the collection of destination's {@link Address}. If {@code null}, it sends to all the members
+    *                     in the cluster.
+    */
+   void sendTo(ReplicableCommand command, DeliverOrder deliverOrder, Collection<Address> destinations);
```

Comment:
I see. In that case I would rather consider `sendTo` and `sendToMany` with the same order of arguments.